libraries/AP_Mount/AP_Mount_SoloGimbal.cpp

```diff
@@ -37,61 +37,69 @@ void AP_Mount_SoloGimbal::update()
         // move mount to a "retracted" position.  we do not implement a separate servo based retract mechanism
         case MAV_MOUNT_MODE_RETRACT:
             _gimbal.set_lockedToBody(true);
-            // initialise _angle_rad to smooth transition if user changes to RC_TARGETTINg
-            _angle_rad = {0, 0, 0, false};
+            mnt_target.target_type = MountTargetType::ANGLE;
+            mnt_target.angle_rad.roll = 0;
```

Comment:
I wonder if we could squeeze down this line like it was before.  Something like, "mnt_target.angle_rad = {0, 0, 0, false};"